docs/docs/headless-cms.md

```diff
@@ -1,9 +1,13 @@
 ---
-title: Sourcing from Headless CMSs
+title: What is a Headless CMS and how to source content from one
 overview: true
 ---
 
-Many content management systems (CMS) now support a “headless” mode, which is perfect for Gatsby sites. A headless CMS allows content creators to manage their content through a familiar admin interface, but allows developers to access the content via API endpoints, allowing for a fully customized frontend experience.
+A *headless CMS* is content management software that enables Writers to produce and organize content, while providing Web Developers with structured data that can be displayed using a separate system on the front-end of a website or app.
```

Comment:
```suggestion
A *headless CMS* is content management software that enables writers to produce and organize content, while providing developers with structured data that can be displayed using a separate system on the front-end of a website or app.
```

I think this doesn't need to capitalized? And since the definition says it can separate the front end of a website _or_ app I think it could just say developers?